frontend/frontend/src/components/RecipeColumn/RecipeColumn.tsx

```diff
@@ -12,6 +12,7 @@ import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
 import FileDownloadOutlinedIcon from '@mui/icons-material/FileDownloadOutlined';
 import FileDownloadDoneOutlinedIcon from '@mui/icons-material/FileDownloadDoneOutlined';
 import ClearOutlinedIcon from '@mui/icons-material/ClearOutlined';
+import CustomModal from '../CustomModal/CustomModal';
```

Comment:
Remove the last "CustomModal" in import path

Reply:
du är CustomModal